plasmapy/utils/decorators/tests/test_validators.py

```diff
@@ -162,40 +152,50 @@ def test_vq_method__get_validations(self):
-            bound_args = sig.bind(*args, **kwargs)
-
-            vq = ValidateQuantities(**case["setup"]["validations"])
-            vq.f = case["setup"]["function"]
-            if "warns" in case:
-                with pytest.warns(case["warns"]):
-                    validations = vq._get_validations(bound_args)
-            elif "raises" in case:
-                with pytest.raises(case["raises"]):
-                    vq._get_validations(bound_args)
-                continue
-            else:
-                validations = vq._get_validations(bound_args)
+        ],
+    )
+    def test_vq_method__get_validations(self, case):
+        # method must exist
+        assert hasattr(ValidateQuantities, "validations")
+        assert hasattr(ValidateQuantities, "_get_validations")
 
-            # only expected argument validations exist
-            assert sorted(validations.keys()) == sorted(case["output"].keys())
+        # setup default validations
+        default_validations = {
+            **self.check_defaults.copy(),
+            "units": [self.check_defaults["units"]],
+        }
 
-            # if validation key-value not specified then default is assumed
-            for arg_name in case["output"].keys():
-                arg_validations = validations[arg_name]
+        sig = inspect.signature(case["setup"]["function"])
+        args = case["setup"]["args"]
+        kwargs = case["setup"]["kwargs"]
+        bound_args = sig.bind(*args, **kwargs)
 
-                for key in default_validations.keys():
-                    if key in case["output"][arg_name]:
-                        val = case["output"][arg_name][key]
-                    else:
-                        val = default_validations[key]
+        vq = ValidateQuantities(**case["setup"]["validations"])
+        vq.f = case["setup"]["function"]
+        if "warns" in case:
+            with pytest.warns(case["warns"]):
+                validations = vq._get_validations(bound_args)
+        elif "raises" in case:
+            with pytest.raises(case["raises"]):
+                vq._get_validations(bound_args)
+            pytest.skip("Skipped raise exception")
+        else:
+            validations = vq._get_validations(bound_args)
+
+        # only expected argument validations exist
+        assert sorted(validations.keys()) == sorted(case["output"].keys())
+
+        # if validation key-value not specified then default is assumed
+        for arg_name in case["output"].keys():
+            arg_validations = validations[arg_name]
+
+            for key in default_validations.keys():
+                if key in case["output"][arg_name]:
+                    val = case["output"][arg_name][key]
+                else:
+                    val = default_validations[key]
 
-                    assert arg_validations[key] == val
+                assert arg_validations[key] == val
 
         # method calls `_get_unit_checks` and `_get_value_checks`
```

Comment:
Delete lines 200-213, since these make up the test method `test_vq_method__get_validations_calls_dependent_methods`.  This test method should look like...

```python
    @pytest.mark.parametrize(
        "_class, method_name",
        [
            (CheckUnits, "_get_unit_checks"),
            (CheckValues, "_get_value_checks"),
        ],
    )
    def test_vq_method__get_validations_calls_dependent_methods(
        self, _class, method_name
    ):
        """
        Test that the `_get_validations` method makes the appropriate
        calls to the `CheckUnits._get_unit_checks` and
        `CheckValues._get_value_checks` methods.
        """
        validations = {
            **self.default_validations,
            "units": [u.cm],
            "can_be_negative": False,
        }
        vq = ValidateQuantities(x=validations)
        vq.f = _foo
        bound_args = inspect.signature(_foo).bind(5)

        with mock.patch.object(_class, method_name, return_value={}) as mock_check_get:

            _ = vq._get_validations(bound_args)

            assert mock_check_get.call_count == 1
            assert mock_check_get.call_args.args[0] == bound_args
```